xml/ha_cluster_lvm.xml

```diff
@@ -117,6 +117,24 @@
      otherwise lvmlockd will fail to start.
     </para>
    </listitem>
+   <listitem>
+    <para>
+     On all nodes, check the value of <parameter>use_lvmlockd</parameter>
+     in <filename>/etc/lvm/lvm.conf</filename>:
+     </para>
+     <screen>&prompt.root;grep 'use_lvmlockd =' /etc/lvm/lvm.conf</screen>
+     <para>
+      If the command returns <literal>use_lvmlockd = 0</literal>, set the
+      parameter's value to <literal>1</literal> on <emphasis>all</emphasis> nodes:
+     </para>
+     <screen>&prompt.root;sed -ie 's/use_lvmlockd = 0/use_lvmlockd = 1/' /etc/lvm/lvm.conf</screen>
+     <para>
+      If this parameter is not enabled, the cluster resources to be configured
+      in <xref linkend="pro.ha.clvm.rsc.lvmlockd" xrefstyle="select:label"/> and
+      <xref linkend="pro.ha.clvm.rsc.vg_lv" xrefstyle="select:label"/> will not
+      start and access to shared VGs or LV will not be granted.
```

Comment:
Shouldn't it be plural?

```suggestion
      start and access to shared VGs or LVs will not be granted.
```

Reply:
Should have been singular here, thanks for catching this!